book/website/collaboration/inclusive-events/inclusive-events-socials.md

```diff
@@ -0,0 +1,146 @@
+(cl-inclusive-events-social-and-networking-guide)=
+# Inclusive Social and Networking Events
+Although your event's main content and sessions will be well thought out, interesting, and engaging for attendees, many attendees often speak highly of the social and collaborative environments that networking events bring to large events. 
+Social events, dinners, and even standing in the lines for coffee can be some of the best times for attendees to get to know one another, identify potential collaborators, and spark ideas.
+
+As an organizer, you should consider how to make these serendipitous opportunities for networking and socializing as accessible as possible for people with lived experience who are attending your event.
+
+Depending on the size and format of your event, this may look like structured networking sessions, a social dinner, or a 15 minute social break at the start or middle of your event. 
+
+Please consider some of the suggestions below, and ensure that you take the time to consider and address any barriers that your specific attendees may face in joining social settings.
+
+(cl-inclusive-events-social-events-timing)=
+## Choosing a Time for Networking 
+If your event schedule allows, ensure that you offer at least one networking or social event during the day. 
```

Comment:
```suggestion
Ideally at least one networking or social event will take place during the event. 
```

Reply:
I messed up reviewing the diff. This has been accepted and added to the file (albeit manually as I hit submit too quickly).